projectX/projectX/Utilities/Extensions.swift

```diff
@@ -136,3 +137,45 @@ func withBoldText(text: String, font: UIFont? = nil) -> NSAttributedString {
   fullString.addAttributes(boldFontAttribute, range: range)
   return fullString
 }}
+
+extension Date {
+    
+  // Function in which calculates the time past on a particular post and returns
+  // this time as a string
+  func diff(date: Date) -> String {
+      
+  // To find the difference in time, we need make a variable to represent the current day
+  let today = Date()
```

Comment:
a more accurate name for this constant would be "timeNow" or "currentTime" as you would be working with units smaller than one day